tests/Unit/DataTransferObjectTypeTest.php

```diff
@@ -303,4 +303,19 @@ public function testNestedDTO(): void
 
         self::assertEquals(['nested' => ['sample_prop' => 'sample']], $data->compact()->toArray());
     }
+
+    public function testGet()
+    {
+        $foo = new Foo;
+        $data = SampleData::make([
+            'simple_prop' => 'foo',
+            'nullable_prop' => 'bar',
+            'mixed_prop' => 'baz',
+            'object_prop' => $foo,
+        ]);
+        self::assertEquals('foo', $data->get('simple_prop'));
+        self::assertEquals('bar', $data->get('nullable_prop'));
+        self::assertEquals('baz', $data->get('mixed_prop'));
+        self::assertEquals($foo, $data->get('object_prop'));
+    }
```

Comment:
```suggestion
    public function testPropertyAccess(): void
    {
        $data = SampleData::make(['simple_prop' => 'foo']);
        
        self::assertSame('foo', $data->get('simple_prop'));
        self::assertSame('foo', $data->simple_prop);
    }
```

Also, it would be nice to have another test for the exception throwing case.

Reply:
@phanan I have add test with exception